test/service/CharitiesRegistrationServiceSpec.scala

```diff
@@ -71,7 +71,7 @@ class CharitiesRegistrationServiceSpec extends SpecBase with BeforeAndAfterEach
       redirectLocation(result) mustBe Some(controllers.routes.RegistrationSentController.onPageLoad().url)
       verify(mockCharitiesConnector, times(1)).registerCharities(any(),any())(any(), any())
       verify(mockUserAnswerRepository, times(1)).set(any())
-      verify(mockAuditService, times(1)).sendEvent(any())(any(), any())
+      verify(mockAuditService, times(2)).sendEvent(any())(any(), any())
```

Comment:
can we verify each audit events using matcher and verify?

Reply:
done - mind that because we're using Mockito I had to hack around the issue, making one test to ensure there were 2 calls of the method and then 2 separate ones to check for the specific parameters called